ui/src/routes/_layout/organizations/$orgId.products.$productId.repositories.$repoId.create-run.tsx

```diff
@@ -987,7 +1008,63 @@ const CreateRunPage = () => {
                 <AccordionItem value='evaluator' className='flex-1'>
                   <AccordionTrigger>Evaluator</AccordionTrigger>
                   <AccordionContent>
-                    No job configurations yet implemented for Evaluator.
+                    <FormField
+                      control={form.control}
+                      name='jobConfigs.evaluator.ruleSet'
+                      render={({ field }) => (
+                        <FormItem>
+                          <FormLabel>Evaluator rules</FormLabel>
+                          <FormControl>
+                            <Input {...field} />
+                          </FormControl>
+                          <FormDescription>
+                            The path to the ruleset file to use for the
```

Comment:
Also here: "rules file".

Reply:
`rulesFile` might be a good variable name, but I think it's better natural language if it's called a rule (singular) file here.

Reply:
Doesn't that somewhat imply that only a single rule can be implemented per file? I'd like to avoid that impression.

Reply:
I don't think that's the implication. There are "rule books" for sports that include a lot of rules, and even closer "rule set" that can include many rules.